tests/setup.test.js

```
@@ -6,21 +6,59 @@ var initialTablesData = require('../fixtures/importMockData.js');
 
 var config = require('../config/load-config.js');
 
-
-test('Server start ok', function (t) {
+function setup (configData, cb) {
   initialTablesData(config, function () {
-    initServer(config, function (err, server, pool) {
-      if (err) {
-        return t.fail('Error starting the server, error: ', err);
+    initServer(config, function (error, server, pool) {
+      if (error) {
+        return cb(false);
```

I think we should return the error on the callback